src/main/java/cc/polyfrost/oneconfig/utils/Notifications.java

```diff
@@ -245,4 +249,27 @@ else if (desiredPosition != entry.getValue().getEnd())
         });
         deltaTime = 0;
     }
+
+    private Animation dummyAnimation;
+    private static final Icon DEFAULT_ICON = new Icon(SVGs.ONECONFIG_HEAD_DARK);
+
+    @Subscribe
+    private void onTickEvent(TickEvent event) {
```

Comment:
register on init if first launch is true, then de register when it is done?

Reply:
why

Reply:
speed; it is not that necessary

Reply:
yeah i was wondering if there was an onLaunch event instead of every tick